supply-chain/catalog/conventions.hbs.md

```diff
@@ -0,0 +1,94 @@
+# Conventions Component
+
+## Description
+
+This document describes the Conventions Component for Tanzu Supply Chains.  The Conventions Component is a way to invoke
+the Conventions Controller in a Tanzu Supply Chain.  Please see the [Conventions Controller](../../cartographer-conventions/about.hbs.md)
+documentation itself for more details on the Conventions Controller service itself.
+
+## API
```

Comment:
**Suggestion:** Can we add the API Schema here?

Reply:
Which API Schema do you mean?  The YAML from the Component definition?

Reply:
I think the default config would be useful

`spec.env` is an array of name, value pairs....

Reply:
Fair enough.  I explained it in the prose below but I will add something more concise in the API section, too.